### PR TITLE
Create MAINTAINERS.md

### PR DESCRIPTION
**Pull request by demarillacizere**
The list of maintainers was provided by G-Research OSS.

### CODE REVIEW

**Review by EnricoMi — COMMENTED** [review]
Alistair contributed to this repo, but is he a maintainer? Is he aware of that?

File: MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## Current maintainers of the project
+
+| Maintainer             | GitHub ID                                               |
+| ---------------------- | ------------------------------------------------------- |
+| EnricoMi               | [EnricoMi](https://github.com/EnricoMi)                 |
```
Comment (EnricoMi) [inline MAINTAINERS.md:5]:
```suggestion
| Enrico Minack          | [EnricoMi](https://github.com/EnricoMi)                 |
```

**Review by EnricoMi — CHANGES_REQUESTED** [review]

**Comment by demarillacizere** [comment]
> Alistair contributed to this repo, but is he a maintainer? Is he aware of that?

I'm not sure, but I can ask the person in charge at G-Research.

**Review by EnricoMi — APPROVED** [review]
Thanks!

**Pull request merged**